apps/board/src/board/board.service.ts

```diff
@@ -10,100 +11,59 @@ export class BoardService {
   public constructor(
     @InjectRepository(Board)
     private readonly boardRepository: Repository<Board>,
-    private readonly dataSource: DataSource,
   ) {}
 
+  @Transactional()
   public async getPosts(): Promise<Board[]> {
-    const queryRunner = this.dataSource.createQueryRunner();
-    await queryRunner.startTransaction();
-    try {
-      const boards = await this.boardRepository.find();
-      await queryRunner.commitTransaction();
-      return boards;
-    } catch (error) {
-      await queryRunner.rollbackTransaction();
-      throw error;
-    } finally {
-      await queryRunner.release();
+    const posts = await this.boardRepository.find();
+
+    if (posts.length === 0) {
+      throw new NotFoundException('게시글이 존재하지 않습니다.');
     }
+
+    return posts;
   }
 
+  @Transactional()
   public async getPostById(id: number): Promise<Board> {
-    const queryRunner = this.dataSource.createQueryRunner();
-    await queryRunner.startTransaction();
-    try {
-      const board = await this.boardRepository.findOneBy({ post_id: id });
-      if (!board) {
-        throw new Error('게시글을 찾을 수 없습니다.');
-      }
-      await queryRunner.commitTransaction();
-      return board;
-    } catch (error) {
-      await queryRunner.rollbackTransaction();
-      throw error;
-    } finally {
-      await queryRunner.release();
+    const post = await this.boardRepository.findOneBy({ post_id: id });
+    if (!post) {
+      throw new NotFoundException('게시글을 찾을 수 없습니다.');
```

Comment:
개행도 일관되게 가져가는 습관을 만들어보시면 좋을 것 같습니다~
```suggestion
    const post = await this.boardRepository.findOneBy({ post_id: id });
    
    if (!post) {
      throw new NotFoundException('게시글을 찾을 수 없습니다.');
```

Reply:
개행을 미처 신경 쓰지 못한 것 같습니다. 감사합니다!
수정된 커밋입니다 : 150862c9d4fa449cc5b838ce5d16be3f1ec573ce